shared/ui/src/commonMain/composeResources/values-fr/strings.xml

```diff
@@ -35,6 +35,7 @@
 
     <string name="faq">FAQ</string>
     <string name="code_of_conduct">Code de conduite</string>
+    <string name="github_repo">Dépôt GitHub</string>
```

Comment:
This feels super weird. I would probably say "GitHub repository" in French too but for the sake of defending the French language, I'm in on "Dépôt GitHub" 🙌